Jenkinsfile

```diff
@@ -66,6 +66,7 @@ pipeline {
     stage("Code Coverage of JavaScript files") {
       steps {
         echo "Run Commmands to get code coverage of JavaScript Files"
```

Comment:
Update echo to say "Running" rather than "Run"

Reply:
Replaced run as running